.github/workflows/ci-conan.yml

```diff
@@ -36,9 +36,19 @@ jobs:
         run: conan remote add osp https://osp-conan.azurewebsites.net/artifactory/api/conan/public --force
       - name: Conan create
         run: |
-          BRANCH=${GITHUB_REF#refs/*/}
-          SHORT_BRANCH=${BRANCH:0:50}
-          CHANNEL=${SHORT_BRANCH//\//_}
+          BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Running this command on a locally checked out tag(v0.4.0) only returns 'HEAD'. Running when checked out the master branch correctly returns 'master'

Reply:
Wow, good catch! I was under the mistaken impression that release builds were also done off `master`, and that what distinguishes them from other builds is whether there's a version tag present on the commit too.

I've pushed a fix that reverts to using `GITHUB_REF` again.